jaxrs-api/src/test/java/jakarta/ws/rs/core/FormTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright must be date of first publication, which is 2024, not 2024. Also, unless you are Oracle, it must mention the original author, which are you.

Reply:
Strictly speaking, the author can consider himself an affiliate, so the copyright is ok. But sure, the original author's name works, too.